src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Formatting/FormattingSpan.cs

```diff
@@ -5,50 +5,17 @@
 
 namespace Microsoft.CodeAnalysis.Razor.Formatting;
 
-internal class FormattingSpan
+internal sealed record class FormattingSpan(
```

Comment:
>record [](http://example.com/codeflow?start=16&length=6)

Out of curiosity, why a record?

Reply:
No specific reason, I just was mindlessly following a "Use primary constructor" message in the error list, and the result seemed like an unnecessarily long file. I just made it a record so the properties would be created for us. Though it does seem like value equality would make the most sense for this type, should anyone ever equate it.